packages/node-cli/messages.json

```diff
@@ -122,6 +126,32 @@
 	"COMMAND_LOCAL_RESOURCE_NOT_FOUND": "The process failed. The {0} file was not found in the project.",
 	"COMMAND_LOCAL_INVALID_XML": "Invalid xml {0}",
 
+	"COMMAND_MANAGE_ACCOUNT_ERROR_CREDENTIALS_EMPTY": "There is no authentication ID (authID) configured for this account-role combination. Run \"suitecloud account:setup\" to set up a NetSuite account to use with SuiteCloud CLI for Node.js.",
+	"COMMAND_MANAGE_ACCOUNT_ERROR_TAG": "The Authentication Tag/Label provided, is not valid, please review the data.",
+	"COMMAND_MANAGE_ACCOUNT_ERROR_UNKNOWN_ACTION": "Unknown selected account:manageauth action",
```

Comment:
Maybe "Unknown selected action for account:manageauth command" to be more readable?